src/store/getters.ts

```diff
@@ -55,6 +55,9 @@ export default {
 			return rootState.conditionRows[ conditionIndex ].propertyValueRelationData.value;
 		};
 	},
+	limit( rootState: RootState ): ( number | undefined ) {
```

Comment:
I feel like I see those parentheses around the types union for the first time, but it seems we used them before, so it's fine :)

Reply:
It's the holidays. :D